internal/feedback/feedback.go

```diff
@@ -104,7 +104,11 @@ func GetLockingFeedback(version, revision, depType, projectPath string) string {
 	}
 
 	if depType == DepTypeImported {
-		return fmt.Sprintf("Trying %s (%s) as initial lock for %s %s", version, revision, depType, projectPath)
+		if version != "" {
+			return fmt.Sprintf("Trying %s (%s) as initial lock for %s %s", version, revision, depType, projectPath)
+		}
+		return fmt.Sprintf("Trying * (%s) as initial lock for %s %s", revision, depType, projectPath)
```

Comment:
It started off as a very different string, and then morphed into this after some discussions with @carolynvs.  Thanks for the fresh eyes and catching this simplification.